addons/crm/models/crm_lead.py

```diff
@@ -1310,10 +1349,111 @@ def _pls_get_naive_bayes_probabilities(self, batch_mode=False):
                     s_lead_lost *= value_result['lost'] / total_lost
 
             # 3. Compute Probability to win
-            lead_probabilities[lead_id] = 100 * s_lead_won / (s_lead_won + s_lead_lost)
+            lead_probabilities[lead_id] = round(100 * s_lead_won / (s_lead_won + s_lead_lost), 2)
         return lead_probabilities
 
+    # ---------------------------------
+    # PLS: Live Increment
+    # ---------------------------------
+    def _pls_reach_state(self, state):
+        """
+        When reaching won (or lost) state, by set won or set lost e.g.,
```

Comment:
I don't understand this part: "by set won or set lost e.g."

Maybe you can simply remove it?